instrumentation/mysql2/lib/opentelemetry/instrumentation/mysql2/patches/client.rb

```diff
@@ -44,6 +44,18 @@ def query(sql, options = {})
 
           private
 
+          def obfuscate_sql(sql)
+            # if ENV variable is set, removes any string values or numbers inside spaces
+            # (to avoid mathcing dates).
+            return sql unless ENV['OTEL_INSTRUMENTATION_MYSQL2_OBFUSCATE']
```

Comment:
```suggestion
            return sql unless ENV['OTEL_RUBY_INSTRUMENTATION_MYSQL2_OBFUSCATE']
```
The OTel spec [recommends](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#language-specific-environment-variables) that language-specific environment variables follow the pattern:
```
OTEL_{LANGUAGE}_{FEATURE}
```

Reply:
Rather than use an env var here (or at least not only an env var), we should add an instrumentation [configuration option](https://github.com/open-telemetry/opentelemetry-ruby/blob/8fbfa71aab426d500e0b3cd9d9ba2a30822e3b7f/api/lib/opentelemetry/instrumentation/base.rb#L140-L154)

Reply:
ok replaced the env var with a config option 👍